docs/source/compile/dynamic_shapes_backed_unbacked.md

```diff
@@ -0,0 +1,44 @@
+# Backed vs Unbacked Symints
+
+Backed `SymInts` are symbolic integers that have a concrete value or "hint"
+associated with them. This means that torch can use these values to make
+decisions about control flow, such as determining which branch of code
+to execute. They are typically derived from operations where the size or
+value is known or can be inferred.
+
+Unbacked `SymInts` are symbolic integers that do not have a concrete value or
+hint. They often arise from data-dependent operations, such as `.nonzero()`
+or `.item()`, where the size or value cannot be determined at compile time.
+Since they lack a concrete value, they cannot be used for control flow
+decisions, and attempting to do so requires a graph break.
```

Comment:
Similar to the 0/1 section, you may also want to call out that unbacked uses size-oblivious semantics. Thus if you are dealing with 0/1 specialization recompilation problems, you may want to explicitly opt-in to unbacked over backed

Reply:
added below: 
Unbacked `SymInts` use *oblivious-size reasoning* which is particularly
useful when you are dealing with
{ref}`0/1 specialization recompilation problem <zero-one-specialization>`.